inferentia/scripts/gen_triton_model.py

```diff
@@ -369,18 +326,19 @@ def create_model_file():
                         type=str,
                         required=True,
                         help='Fullpath to the compiled model')
-    parser.add_argument('--avbl_neuron_cores_count',
-                        type=int,
-                        default=4,
-                        help='The number of available neuron cores')
+    parser.add_argument('--neuron_core_range',
```

Comment:
We should set a default core range. Maybe just the first core?

Reply:
I would want the user to be aware of this option and not silently introduce the limited executions..